gear/user_management/src/python/user_app/run.py

```diff
@@ -189,8 +189,11 @@ def __get_user_queue(self, user_file_path: str) -> UserQueue[UserEntry]:
         user_list: UserQueue[UserEntry] = UserQueue()
         for user_doc in object_list:
```

Comment:
Add a check to skip entries with `permissions_approval = False`

Reply:
those are already filtered out at this point, but I did add another check after the object is loaded.